OpenRA.Mods.D2k/Activities/SwallowActor.cs

```diff
@@ -103,9 +101,9 @@ public override bool Tick(Actor self)
 					stance = AttackState.Burrowed;
 					countdown = swallow.Info.AttackDelay;
 					burrowLocation = self.Location;
-					if (conditionManager != null && attackingToken == ConditionManager.InvalidConditionToken &&
+					if (attackingToken == Actor.InvalidConditionToken &&
```

Comment:
This could be brought back onto one line now, IMO.

Reply:
Done.